.env

```diff
@@ -0,0 +1,3 @@
+TYPE=development
+API_HOST=https://api.staging.sample.com
+TEST=dev
```

Comment:
Hmm you can remove TEST variable, becase it is doing the same as TYPE variable, so it saying  what enviroment we are using.